<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Garamond Pro Bold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-width:4px;
border-radius:10px;
color:black;
border:1px solid gray;}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Garamond Pro</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-width:4px;
border-radius:10px;
color:black;
border:1px solid gray;}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
     background-color: rgba(47,79,79, 200);/*背景色*/
     border-style: outset; /*边框样式*/
     border-width: 4px;  /*边框宽度为4像素*/
     border-radius: 10px; /*边框圆角半径*/
     border-color: rgba(255, 225, 255, 30);/*边框颜色*/
     font: bold 20px;/*字体*/
         color:rgba(255, 255,255, 200);/*字体颜色*/
     padding: 10px; /*填衬*/
 	
}

QPushButton:hover{ /*鼠标悬停在按钮上时*/
	background-image: url(:res/super.gif);
	background-repeat: no-repeat;
	background-position:left;
        background-color:rgba(100,255,100, 100);
        border-color: rgba(255, 225, 255, 200);
        color:rgba(0, 0, 0, 200);
		 
}

QPushButton:pressed { /*按钮被按下时*/
     background-color:rgba(100,255,100, 200);
         border-color: rgba(255, 225, 255, 30);
     border-style: inset;
        color:rgba(0, 0, 0, 100);
}</string>
   </property>
   <property name="text">
    <string>YES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="concel_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
     background-color: rgba(47,79,79, 200);/*背景色*/
     border-style: outset; /*边框样式*/
     border-width: 4px;  /*边框宽度为4像素*/
     border-radius: 10px; /*边框圆角半径*/
     border-color: rgba(255, 225, 255, 30);/*边框颜色*/
     font: bold 20px;/*字体*/
         color:rgba(255, 255,255, 200);/*字体颜色*/
     padding: 10px; /*填衬*/
 	
}

QPushButton:hover{ /*鼠标悬停在按钮上时*/
	background-image: url(:res/super.gif);
	background-repeat: no-repeat;
	background-position:left;
        background-color:rgba(100,255,100, 100);
        border-color: rgba(255, 225, 255, 200);
        color:rgba(0, 0, 0, 200);
		 
}

QPushButton:pressed { /*按钮被按下时*/
     background-color:rgba(100,255,100, 200);
         border-color: rgba(255, 225, 255, 30);
     border-style: inset;
        color:rgba(0, 0, 0, 100);
}</string>
   </property>
   <property name="text">
    <string>CONCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border-image:url(:/login.png)}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
